views/default/settings.pdt

```diff
@@ -16,6 +16,14 @@
 		</li>
 	</ul>
 
+    <div class="title_row">
+        <h3><?php $this->_('StripePayments.webhook'); ?></h3>
+    </div>
+    <div class="pad">
+        <p><?php $this->_('StripePayments.webhook_note'); ?></p>
+        <code><?php echo Configure::get('Blesta.gw_callback_url') . Configure::get('Blesta.company_id') . '/stripe_payments/'; ?></code>
```

Comment:
Currently, the callback only handles nonmerchant gateways

Reply:
I created a PR for the core, adding support for merchant gateways.

Reply:
Got it.  The link generated by this uses /callback/gw instead of /calback/mgw